pkgs/development/compilers/rust/cargo.nix

```diff
@@ -1,14 +1,15 @@
 { lib, stdenv, pkgsBuildHost, pkgsHostHost
 , file, curl, pkg-config, python3, openssl, cmake, zlib
-, installShellFiles, makeWrapper, rustPlatform, rustc
+, installShellFiles, makeWrapper, rustPlatform, rust, rustc
 , CoreFoundation, Security
-, auditable ? true
+, auditable ? !cargo-auditable.meta.broken
 , cargo-auditable
+, pkgsBuildBuild
 }:
 
-rustPlatform.buildRustPackage.override {
+rustPlatform.buildRustPackage.override (lib.optionalAttrs (!cargo-auditable.meta.broken) {
   cargo-auditable = cargo-auditable.bootstrap;
-} {
+}) ({
```

Comment:
is this necessary? I was able to build `pkgsCross.aarch64-multiplatform.cargo` without this

Reply:
Yep, you're right: 83ccdd2d92fa